.github/workflows/release-please.yml

```diff
@@ -26,3 +29,100 @@ jobs:
             VERSION
             barretenberg.nix
             barretenberg-wasm.nix
+
+  build-linux:
+    name: Build on Ubuntu
+    needs: [release-please]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install Bleeding edge CMake
+        run: |
+              sudo apt -y remove --purge cmake
+              sudo snap install cmake --classic
+      - name: Install Yarn
+        run: |
+          curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+          echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+          sudo apt -y update && sudo apt -y install yarn
+      - name: Install Clang/Ninja/Binaryen
+        run: |
+          sudo apt-get -y install clang clang-format ninja-build binaryen
+      - name: Install WASI-SDK
+        run: |
+          cd cpp
+          ./scripts/install-wasi-sdk.sh
+      - name: Compile Barretenberg
+        run: |
+          cd cpp
+          cmake --preset default -DCMAKE_BUILD_TYPE=RelWithAssert -DOMP_MULTITHREADING=OFF
+          cmake --build --preset default ${@/#/--target }
+      - name: Compile Barretenberg.js
+        run: |
+          cd ts
+          yarn && yarn build
+      - name: Tar Typescript `dest` folder
+        run: tar -cvf dest.tar ts/dest
+      - name: Tar Native static `lib` folder
+        run: tar -cvf ubuntu-static-lib.tar cpp/build/lib
```

Comment:
How come outputting static libs? Were we not going to output the artefects discussed in the thread?